src/pages/Home.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import HomeNavbar from '../components/HomeNavbar';
+import Region from '../components/Region';
+
+const Home = () => {
+  const data = useSelector((state) => state.data);
+
+  const [search, setSearch] = useState('');
+
+  const handleChange = (event) => setSearch(event.target.value);
+
+  const filteredRegions = data.regions.sort((a, b) => b.today_open_cases - a.today_open_cases)
+    .filter((region) => region.name.toLowerCase()
+      .includes(search.toLowerCase()));
+
+  return (
+    <>
+      <HomeNavbar
+        search={search}
+        handleChange={handleChange}
+      />
+      <div className="main-container flex-box">
```

Comment:
- [ ] For better sematic, I think a `section` tag would serve better here. :pray:

Reply:
Required change completed.